.changeset/gorgeous-adults-divide.md

```diff
@@ -0,0 +1,8 @@
+---
+"@internal/fuel-core": patch
```

Comment:
There is no need to add the internal packages into the changeset, and if it's not possible without adding it we should fix that.